neovim/plugin/script_host.py

```diff
@@ -63,6 +63,12 @@ def setup(self, nvim):
         self.saved_stderr = sys.stderr
         sys.stdout = RedirectStream(lambda data: nvim.out_write(data))
         sys.stderr = RedirectStream(lambda data: nvim.err_write(data))
+        # Setup the DirChanged listener
+        cid = nvim.channel_id
+        cmd = 'autocmd DirChanged *' \
+            + ' call rpcrequest(%d, "python_chdir", v:event)' \
```

Comment:
also we prefer to use new-style `.format()`

Reply:
I, too, prefer `.format()`. I simply followed what I've seen being used before. Will change it.